app/middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const { badRequestError } = require('../errors');
+const logger = require('../logger');
+const { verifyToken } = require('../utils/jwt');
+
+exports.authenticated = (req, res, next) => {
+  try {
+    const accessToken = req.headers.authorization;
+
+    if (!accessToken) throw new Error();
+
+    const payload = verifyToken(accessToken);
+
+    const { user: id, name } = payload;
+    if (!id || !name) throw new Error();
+
+    res.locals.user = { id, name };
```

Comment:
Why do you add it to the `res`? It should be to `req` and something simpler like `req.user`